src/richie/apps/courses/templates/courses/plugins/organization_plugin.html

```diff
@@ -0,0 +1,52 @@
+{% load cms_tags i18n %}
+{% spaceless %}
+{% if current_page.publisher_is_draft %}
+  {% if instance.organization.check_publication is True %}
+    <div class="organization-plugin-container">
+      <div class="organization-plugin">
+        <a class="organization-plugin__body" href="{{ instance.organization.public_extension.extended_object.get_absolute_url }}"
```

Comment:
I need to check the presence of this tag in 3 parts. Otherwise the test fails. I'm not sure why.